### PR TITLE
Code for Lesson 29

### PR DESCRIPTION
**Pull request by Lrenicus**
I had alot of fun on the exercise. That last problem took a lot of reading and understanding of the .map(). @Nehero

### CODE REVIEW

File: lessons/29-js-arrays/index.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        console.log('You still need to read ' + bookIf);
+    }
+}
+
+
+//(Difficult) - Recreate the array.map function. Complete the following code
+
+function map(arr, func) {
+    var newArr = arr.map(func);
+    return newArr;
```
Comment (ozziexsh) [inline lessons/29-js-arrays/index.js]: Happy you tried it! While this works, try and recreate the _functionality_ of map without actually using `array.map()`. 

Try and use a normal loop in here to create a new array of values

File: lessons/29-js-arrays/index.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+//recipe card
+
+var favRecipe = {
+    title: 'Mole',
+    servings: 2,
+    ingredients: ["cinnamon", "cumin", "cocoa"],
+}
+
+console.log(favRecipe.title);
+console.log('serves: ${favRecipe.servings}');
```
Comment (ozziexsh) [inline lessons/29-js-arrays/index.js]: In order to use a variable inside of a string these would need to be backticks 

```javascript
console.log(`serves: ${favRecipe.servings}`);
```

File: lessons/29-js-arrays/index.js (hunk 1 above)
Comment (Lrenicus) [reply lessons/29-js-arrays/index.js]: My research on the array.map() came in handy as i already knew what it did. recreating it in a loop was fun. @nehero

File: lessons/29-js-arrays/index.js (hunk 2 above)
Comment (Lrenicus) [reply lessons/29-js-arrays/index.js]: Thank you, fixed.

File: lessons/29-js-arrays/index.js (hunk 1 above)
Comment (ozziexsh) [reply lessons/29-js-arrays/index.js]: Boom ✊

**Pull request merged**